server/tools/peertube-import-videos.ts

```diff
@@ -68,30 +106,37 @@ async function run (url: string, user: UserInfo) {
   const options = [ '-j', '--flat-playlist', '--playlist-reverse' ]
   youtubeDL.getInfo(program[ 'targetUrl' ], options, processOptions, async (err, info) => {
     if (err) {
-      console.log(err.message)
+      log.error(err.message)
       process.exit(1)
     }
 
     let infoArray: any[]
 
     // Normalize utf8 fields
     if (Array.isArray(info) === true) {
-      infoArray = info.map(i => normalizeObject(i))
+      if (program[ 'first' ]) {
+        infoArray = info.slice(0, program[ 'first' ]).map(i => normalizeObject(i))
```

Comment:
Shouldn't it be `info.slice(program[ 'first' ])`?

Reply:
The starting zero is there for clarity, it is optional according to the specs.